### ISSUE TITLE
Why don't use softmax before cross entropy in multibox loss?

### ISSUE THREAD

**Issue by Kaidy-Boom** [amdegroot/ssd.pytorch#218]
Hi,
 
I'm recently doing some modifying on your code. I found that you didn't use softmax before cross entropy in multibox loss which has been used in other SSD implements and alse mentioned in the paper. 

However, I tried change it and re-train model without any pre-trained model on VOC 2007. The loss log confused me that softmax makes it converge slowly and seems worse than no-softmax. Can you explain why you remove softmax?  Thx.

**Comment by AllenMao** [comment]
you can see [here](https://pytorch.org/docs/0.3.1/_modules/torch/nn/functional.html#cross_entropy), torch.nn.functional.cross_entropy() combines log_softmax and nll_loss in a single function.
